browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1003,6 +1003,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 				var isSelectedTab = data.selected == item.id;
 				if (isSelectedTab) {
 					$(tab).addClass('selected');
+					tab.title = _('Tap to collapse');
```

Comment:
this will be added not only in notebookbar but in all the dialogs...

it should be added to the JSON in https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.NotebookbarBuilder.js#L377
I would add that to every item as 'tooltip'
{
    ...
    tabs: [ ... { id: ..., number:... ,   tooltip: _('Tap to collapse'), ... }, ... ],
    ...
}
and here we just use that value

Reply:
also we need to change `_createTabClick` function above so we will get the tooltip not only for the tab which initially was selected, but after we activate other tab also

Reply:
Will do